torch/csrc/jit/codegen/cuda/lower_alias_memory.cpp

```diff
@@ -851,50 +925,61 @@ void BufferReuseDebugPrinter::printAllocInfo(const kir::Allocate* alloc) {
   if (alloc_info->can_use_inner_alias) {
     os_ << "inner live interval: ";
     os_ << alloc_info->inner_live_interval->toString() << " , ";
+  } else {
+    os_ << "cannot use inner alias, ";
   }
   os_ << "size expr : " << alloc_info->size_expr << " , "
       << "outer live interval: " << alloc_info->outer_live_interval->toString();
   indent() << message_end;
 }
 
 //! Reuse Allocation nodes via pointer aliasing
-class AllocateReuseModifier {
```

Comment:
This class is also decomposed into two passes in this PR. The first pass is to find allocations that can be aliased to other allocations. The second pass is to replace the identified allocations to alias allocations. The first pass is done by `ReusableAllocationFinder`, while the second pass is done by `AllocationReuseModifier`.